lang/nl.json

```diff
@@ -64,6 +61,8 @@
     "Thank you for subscribing": "Bedankt voor uw inschrijving",
     "Thank you for your order": "Bedankt voor uw bestelling",
     "This is a business account": "Dit is een zakelijke account",
+    "This product is out of stock": "Dit product is op dit moment niet op voorraad",
+    "This product it out of stock, remove it to continue your order.": "Dit product is niet op voorraad, verwijder het om verder te gaan met je bestelling.",
```

Comment:
Uw / Je translations

Reply:
"u" is to formal here, let's stick to "je" for everything.